formal/data_ind_timing/Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# A simple wrapper around fusesoc to make it a bit easier to run the formal flow
+
+core-name   := lowrisc:fpv:ibex_data_ind_timing
+vlnv        := $(subst :,_,$(core-name))
+build-root  := $(abspath ../../build/$(vlnv))
+
+# Since we have a hacky hook that runs sv2v in place on fusesoc's
+# copied source files, we have to generate different build roots for
+# the two flavours (otherwise bad things will happen if you run make
+# -j2)
+mk-build-root = $(abspath ../../build/$(vlnv)/$(1))
+
+mk-fusesoc-cmd = \
+  fusesoc --cores-root=../.. \
+  run --build-root=$(call mk-build-root,$(1)) \
+  --target=$(1) \
+  $(core-name) $(fusesoc-params)
+
+.PHONY: all slow fast single
+all: slow fast single
+
+slow: slow_mull slow_mulh slow_div slow_rem
+fast: fast_mull fast_mulh fast_div fast_rem
+single: single_mull single_mulh single_div single_rem
+
+slow_mull slow_mulh slow_div slow_rem:
+	$(call mk-fusesoc-cmd,$@)
+fast_mull fast_mulh fast_div fast_rem:
+	$(call mk-fusesoc-cmd,$@)
+single_mull single_mulh single_div single_rem:
+	$(call mk-fusesoc-cmd,$@)
```

Comment:
Yes, should be possible to do that.
I repeated the code only because I did not want to have a long line with all the targets. But I did not have the idea to generate the targets as you suggested. Will give it a go, thanks.